### ISSUE TITLE
Botania Flowers spawning in your dimension(s).

### ISSUE THREAD

**Issue by chemicalvamp** [Y3Z0N/theabyss#22]
### Describe the Suggestion

I really like this radio dream world, But Botania flowers shouldn't be spawning in it, right? Is there a way for me to blacklist them on my end?

I haven't yet tested the other dimensions of yours yet.

**Comment by Y3Z0N** [comment]
this "bug" is caused by Botania it self. it seems like vazkii don't have a config to disable it in custom dimensions. please report this issue to vazkii (botania dev)

**Comment by chemicalvamp** [comment]
Yes, I have thanks for that. And nice work! The first two dreams were great, I hope I see more.
